### PR TITLE
Add SQLitePCLRaw 1.1.6 to allowed third-party dependencies

### PR DESCRIPTION
**Pull request by natemcmaster**
This version of SQLitePCLRaw fixes issues with netstandard2.0 and AssetTargetFallback.

cc @Eilon  - OSS approval

### CODE REVIEW

File: src/NuGetPackageVerifier/third-party.json [hunk 1]
```diff
@@ -41,11 +41,11 @@
...
-    "SQLitePCLRaw.bundle_green": [ "1.1.3", "1.1.5" ],
-    "SQLitePCLRaw.core": [ "1.1.3", "1.1.5" ],
-    "SQLitePCLRaw.lib.e_sqlite3.v110_xp": [ "1.1.3", "1.1.5" ],
-    "SQLitePCLRaw.lib.e_sqlite3.linux": [ "1.1.3", "1.1.5" ],
-    "SQLitePCLRaw.lib.e_sqlite3.osx": [ "1.1.3", "1.1.5" ],
+    "SQLitePCLRaw.bundle_green": [ "1.1.3", "1.1.5", "1.1.6" ],
+    "SQLitePCLRaw.core": [ "1.1.3", "1.1.5", "1.1.6" ],
+    "SQLitePCLRaw.lib.e_sqlite3.v110_xp": [ "1.1.3", "1.1.5", "1.1.6" ],
+    "SQLitePCLRaw.lib.e_sqlite3.linux": [ "1.1.3", "1.1.5", "1.1.6" ],
+    "SQLitePCLRaw.lib.e_sqlite3.osx": [ "1.1.3", "1.1.5", "1.1.6" ],
```
Comment (bricelam) [inline src/NuGetPackageVerifier/third-party.json]: Do we need the closure or can the *.lib.* ones be removed?

Comment (natemcmaster) [inline src/NuGetPackageVerifier/third-party.json]: Not sure. @pakrym does this list flatten dependencies? Or does it only check explicit `PackageReference`'s?

Comment (pakrym) [inline src/NuGetPackageVerifier/third-party.json]: It only checks direct `PackageReference`'s

Comment (natemcmaster) [inline src/NuGetPackageVerifier/third-party.json]: Ok. We can reduced to just .core and .bundle_green after https://github.com/aspnet/MetaPackages/pull/164

Comment (JunTaoLuo) [inline src/NuGetPackageVerifier/third-party.json]: Do we need the previous versions? Shouldn't only 1.1.6 be needed?

Comment (natemcmaster) [inline src/NuGetPackageVerifier/third-party.json]: If the tool versioned with our code, I would delete old versions. But in theory old builds of the code might be built with new versions of the tool. Ideally this config data won't ship in the tool. #266

**Comment by natemcmaster** [comment]
Closing. This version of SQLite bundled in this update appears to includes a regression in LEFT JOIN. Let's not update until we have a fix or a different bundled version.

**Pull request closed without merging**